### DISCUSSION TITLE
Ant select dropdown width issue

### DISCUSSION THREAD

**Discussion by truonghoit** [ant-design/ant-design#28661] [Q&A]
I'm facing a small issue about select dropdown box, the width of the dropdown and the input is different a bit, it makes the ui become quite strange, like this: 
![image](https://user-images.githubusercontent.com/52435448/103514128-9960f680-4e9e-11eb-91cb-985e9588596c.png)

It happens when I set border 2px, on many screen sizes. (like: 1440 x 1160). 
Follow the link for the detail: https://codesandbox.io/s/antd-reproduction-template-forked-puqwl?file=/index.css
Is anyone in the same situation?

**Comment by xrkffgg** [comment]
Hello, please provide a online reproduction by forking this link https://u.ant.design/codesandbox-repro or a minimal GitHub repository.

**Reply by truonghoit** [reply]
This is my link: https://codesandbox.io/s/antd-reproduction-template-forked-puqwl?file=/index.css

It happens on many screen sizes. (like: 1440 x 1160)
